automatic/mp3tag/legal/LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Mp3tag v2.91 Copyright �1999-2018 Florian Heidenreich
```

Comment:
Seems like an invalid character came into existence, did you save this using the `Windows 1252` encoding instead of UTF8?

Please update the file to be utf8 (without BOM) encoded when it's saved.